README.md

```diff
@@ -43,7 +43,9 @@ Options:
   --servers        include complete servers object with --valid        [boolean]
   --lineWidth, -l  max line width of yaml output          [number] [default: -1]
   --maxAliasCount  maximum YAML aliases allowed          [number] [default: 100]
+  --configFile     The file & path for the filter options                 [path]
   --help           Show help                                           [boolean]
+  --verbose        Output more details of the filter process           [boolean]
```

Comment:
`verbose` should be a yargs `count` type not a boolean, so you can do `--verbose --verbose [...]` to get increasing levels of verbosity.